inc/commonitiltask.class.php

```diff
@@ -1719,6 +1719,13 @@ function showMassiveActionAddTaskForm() {
       echo "</td>";
       echo "</tr>";
 
+      echo "<tr class='tab_bg_2'>";
+      echo "<td>".__('Status')."</td>";
+      echo "<td>";
+      Planning::dropdownState("state", $_SESSION['glpitask_state']);
```

Comment:
See https://github.com/glpi-project/glpi/blob/9.4/bugfixes/inc/commonitiltask.class.php#L526 . Is it possible that this session value is not available ?

Reply:
in the case of `post_getEmpty`, yes i think it's possible (cron, units tests, etc).
In case of displaying a massive action : no, the session var will be always set.